static/js/taxpayer_creation.js

```diff
@@ -4,12 +4,64 @@ var left, opacity, scale;
 var animating; 
 
 
-$(".next").click(function(){
+// Checks if first part of form is complete
+function check_first_fieldset_complete(){
+	var token = true
+	$(".mandatory>input:text, .mandatory>input[type='number'], .mandatory>select, .mandatory>input[type='email']").each(function(index){
```

Comment:
maybe we can first name this selector to something representative to know what it is and then execute the 'each' loop. i.e `let selector_name = $(".mandatory>input:text, .mandatory>input[type='number'], .mandatory>select, .mandatory>input[type='email']")`